composer.json

```diff
@@ -30,16 +30,32 @@
     ],
     "require": {
         "php": ">=5.6",
-        "symfony/event-dispatcher": "^2.7|^3.0|^4.0",
+        "symfony/event-dispatcher": "^2.7|^3.0|^4.0|^5.0",
```

Comment:
Should be `^4.3|^5.0` because the `LegacyEventDispatcherProxy` does not exist prior to that, and the Symfony 5 version of it does nothing so it does not support older parameter order. So it's impossible to support both Sf5 and anything prior to 4.3 at the same time, AFAIK.

Reply:
Hello, you are right thank you for the information. 

I created these MRs because tests of #329 was not passing, but it looks that mine don't pass either! So i closed them. 

I saw that you open a MR that resolve all these issues, it's cool, thank you.

Reply:
It was very tricky because the classes override the Symfony ones, but these are not consistent across all Symfony versions so it makes it difficult to support all versions without breaking backwards compatibility for this specific lib. But turns out it's not impossible actually.